### PR TITLE
atari.sgml: Add documentation for Atari clock_... functions implementation

### PR DESCRIPTION
**Pull request by groessler**

### CODE REVIEW

File: doc/atari.sgml [hunk 1]
```diff
@@ -634,7 +634,18 @@ interface module.
...
 <sect>Limitations<p>
 
-<sect1><tt/atarixl/<#if output="info|latex2e"> limitations</#if><label id="limitations"<p>
+<sect1><tt/Realtime clock/<label id="realtimeclock"<p>
+
+Access to the real time clock is supported only when running on SpartaDOS-X.
+There needs to be an RTC driver installed. This is normally the case
+in the default installation (CONFIG.SYS) of SpartaDOS-X.
+A missing RTC driver in SpartaDOS-X is not supported, and the program
+may crash when calling the <tt/clock_settime()/ or <tt/clock_readtime()/
```
Comment (greg-king5) [inline doc/atari.sgml]: Typo: `clock_readtime` -> `clock_gettime()`

Comment (groessler) [reply doc/atari.sgml]: fixed, thanks

File: doc/atari.sgml [hunk 2]
```diff
@@ -634,7 +634,18 @@ interface module.
 
 <sect>Limitations<p>
 
-<sect1><tt/atarixl/<#if output="info|latex2e"> limitations</#if><label id="limitations"<p>
+<sect1><tt/Realtime clock/<label id="realtimeclock"<p>
+
+Access to the real time clock is supported only when running on SpartaDOS-X.
+There needs to be an RTC driver installed. This is normally the case
```
Comment (oliverschmidt) [inline doc/atari.sgml]: I deliberately stayed clear from the abbreviation 'RTC' in the docs for the Apple II and the C64. Maybe you want to adhere to that here too.

**Comment by oliverschmidt** [comment]
> Hmm, added by mistake some other changes, to gamate and telestrat. But they should be good.

Thanks for the hint.

**Pull request merged**